ts/graphs/CardCounts.svelte

```diff
@@ -124,8 +124,15 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         }
     }
 
-    .search-link:hover {
+    .search-link {
+        border: none;
+        background: transparent;
         cursor: pointer;
+        box-shadow: none;
+        padding: 4px;
```

Comment:
Do we need the padding? The text looks a bit too spaced out now, compared to the other graphs around it.

Reply:
Kenny is fine.

Regarding the spacing, I added the padding so that the text has some breathing room when tabbed. It ensures it isn't squished with the focus outline.

I can fix the excessive spacing by modifying the margin value to match the existing design. Here is how it looks now.

<img width="487" alt="Screenshot 2023-07-20 at 15 35 11" src="https://github.com/ankitects/anki/assets/32024147/07efc1f3-a50b-4c87-9f24-4f8a8a059250">

Reply:
Looks good - thanks Kenny!